semanticdb/semanticdb3/semanticdb3.proto

```diff
@@ -44,7 +44,10 @@ message SymbolInformation {
     OBJECT = 10;
     PACKAGE = 11;
     PACKAGE_OBJECT = 12;
-    CLASS = 13;
+    // TODO: Can't say CLASS here, because of a Scala Native bug.
+    // Unlike other todos in this pull request, this one doesn't have a workaround.
+    // https://github.com/scala-native/scala-native/issues/1155.
+    CLAZZ = 13;
```

Comment:
@densh This is a hard blocker, because this name is part of our API. Do you think you could help us out with this issue? If you don't have time for that, can you give some pointers so that I can fix the issue myself?

Reply:
https://github.com/scala-native/scala-native/pull/1156